.github/renovate.json

```diff
@@ -35,34 +35,62 @@
     "**/Dockerfile.template"
   ],
   "repositories": [
+
     "balena-io-modules/open-balena-base",
+    "balena-io-modules/scrutinizer",
+
+
+    "balena-io-examples/balena-python-hello-world",
 
+
+    "balena-io/analytics-backend",
     "balena-io/autohat",
+    "balena-io/balena-admin",
     "balena-io/balena-alertmanager",
+    "balena-io/balena-api",
+    "balena-io/balena-builder",
     "balena-io/balena-cloud",
+    "balena-io/balena-delta",
+    "balena-io/balena-git",
+    "balena-io/balena-img",
     "balena-io/balena-mdns-publisher",
     "balena-io/balena-metrics-guard-base",
     "balena-io/balena-monitor",
+    "balena-io/balena-proxy",
+    "balena-io/balena-sentry",
     "balena-io/balena-sentry",
+    "balena-io/balena-ui",
```

Comment:
@ab77 I'm a bit concerned about adding the UI right away, since the only automated test we have atm is that the login page shows up.

Reply:
@thgreasi just take it out then, idk if it's possible to exclude a repo.